src/components/JobList/index.tsx

```diff
@@ -44,14 +44,12 @@ function JobList() {
       }
       <div className="jobs-preview" data-testid="job-preview">
         {selectedJob && <JobContentPreview
-
           title={selectedJob.title}
           company={selectedJob.company}
           description={selectedJob.description}
           requirement={selectedJob.requirements}
           benefits={selectedJob.benefits}
           salary={selectedJob.salary}
-          createAt={selectedJob.createdAt}
```

Comment:
Hi, Why do we need to remove this `createAt` ?

Reply:
hello, i think this project is basic so i don'n use createAt, so we should use it for this project?

Reply:
yes please, should keep it as original

Reply:
I fixed it and commit the project. I hope you check it out for me. thanks

Reply:
ok cool